examples/haitham_work_area/genzsprtwo_37_dir_organizer/genz_sprtwo_37_dir_org.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shutil
+import glob
+
+def organize_files(folder):
```

Comment:
There might be cases when we would have to use this method in any other .py module. If that happens, it would be really hard to use it, since it just receives a folder argument and then categories are hardcoded and not received as argument.
A suggestion is to refactor the code so it becomes usable in other cases, for example:
If a person wants to organize .css and .js files in css and js dir then this function can not be used.